2.6.md

```diff
@@ -695,7 +695,7 @@ There are also several subordinate objects that provide detailed data to potenti
   <tr>
     <td><code>id</code></td>
     <td>string; required</td>
-    <td>Unique ID of the bid request, provided by the exchange.</td>
+    <td>Globally unique ID of the bid request, provided by the exchange. Pass-through seller platforms receiving this id should generate a new id when sending requests to multiple entities.</td>
```

Comment:
Thanks @patmmccann. Please upper-case all the references to ID for consistency.

I'm inclined to strike the new sentence, in part because now that needs to be qualified (are we prescribing sending the same new ID to all the multiple entities, what if there is only one entity, etc).

It should be enough to say the field value belongs to the exchange. If it helps, we may illustrate why uniqueness is important -- e.g., it's required for the exchange to be able to:
- Join bidder responses to opportunities from this request, at auction time.
- Join impression events to data about those opportunities, at render time.

Reply:
I find the current wording of the spec ambiguous as to wether the exchange should pass the same request id as-is downstream or if it's appropriate to generate a new request id when "fanning out" requests. This led to discussions of how the request id differs from the transaction id. As we discussed in the IAB Working Group, it's appropriate for the exchange to assign a unique id when passing the request to each downstream system.

Perhaps we could iterate on the wording. How about something like:
"ID of the bid request, assigned by the exchange, unique to each downstream advertising system."

Reply:
> I'd like to suggest we iterate on the exact text.

Thanks, 100%. I'll have a go too :-) how about this? "ID of the bid request, assigned by the exchange, and unique for the exchange's subsequent tracking of the responses".

I'm only trying to keep this as simple as possible without losing specificity. I don't know that it's necessary to prescribe sending different values to different recipients of the same event (although it can be wise). Plus people disagree about which side of the exchange is downstream. ;-)

Reply:
adopted! How does it look now?

Reply:
Thanks @patmmccann @SyntaxNode looks good. 👍